Changelog.md

```diff
@@ -1,3 +1,19 @@
+## 4.0.0-beta7 - UNRELEASED
```

Comment:
Worth reviewing wording with @greg-fer and maybe additionally highlighting these changes :)